SolderingPen_ESP32S2/SolderingPen_ESP32S2.ino

```diff
@@ -686,25 +688,49 @@ void MainScreen() {
     u8g2.setFontPosTop();
     //    u8g2.drawUTF8(0, 0 + SCREEN_OFFSET, "设温:");
     u8g2.drawUTF8(0, 0 + SCREEN_OFFSET, txt_set_temp[language]);
-    u8g2.setCursor(40, 0 + SCREEN_OFFSET);
+    if(language == 2)
+      u8g2.setCursor(8, 0 + SCREEN_OFFSET);
```

Comment:
```suggestion
      u8g2.setCursor(36, 0 + SCREEN_OFFSET);
```
I tried to keep name `SET:` with offset `36` and it looks more or less separated with 5 letters words.

![image](https://github.com/Eddddddddy/Songguo-PTS200/assets/8087963/d377f69b-6f60-41e7-baaf-6e1a98055d29)

Reply:
It is. Kinda still separated, you are right.